<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSMM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MapDescriptorWidget" name="tableWidget"/>
    </item>
    <item>
     <widget class="QWidget" name="mapToolbar" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addMap">
         <property name="text">
          <string>Add Map</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeMap">
         <property name="text">
          <string>Remove Map(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_WBFS_ISO"/>
    <addaction name="actionExport_To_Folder"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Optional Patches</string>
    </property>
    <addaction name="actionPatch_Wiimmfi"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCSMM_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCSMM_Help">
   <property name="text">
    <string>CSMM Help</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Game Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_To_Folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export To Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPatch_Wiimmfi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Patch Wiimmfi</string>
   </property>
  </action>
  <action name="actionImport_WBFS_ISO">
   <property name="text">
    <string>Import WBFS/ISO</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapDescriptorWidget</class>
   <extends>QTableWidget</extends>
   <header>mapdescriptorwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
